docs/infra/Packages-Gardener-Rotation.md

```diff
@@ -0,0 +1,143 @@
+
+Team members are not expected to participate in multiple Flutter rotations. For example, Flutter framework gardeners are exempt and vice versa. New team members should be added to a single rotation.
+
+Before heading out on holiday, or if you get to your shift and find you can't do it, check the upcoming rotations and find a volunteer to swap shifts with while you're out. During some holiday periods when many team members are out and activity is particularly low on the tree, it may not be essential to have a dedicated gardener.
+
+### Journal
+
+Please briefly describe any issues you encountered during your week of gardening in the [packages gardening journal], along with any solutions you think may be useful for other gardeners in the future!
+
+## Periodic scan
+
+Below are the tasks that should be done routinely while gardening.
+
+### Dashboard
+Open the [packages build dashboard].
+1. If the tree is closed, identify which test shards are failing. If there are yellow boxes with an exclamation point, that means that the failed tests are automatically re-running themselves. The tree is not fully closed until there are solid red boxes or red boxes with exclamation points. You can begin investigation as soon as you notice the tree going red, but it is suggested not to begin escalation until re-runs have completed.
+1. Identify which test within the shard failed, and try to locate obvious errors or failures in the logs.
+1. Update the [hackers-ecosystem channel] on [Discord] with an announcement that the tree is red, and that you are working on it.
+1. Cross reference the failure with the commit in question. If it is obvious the PR in question caused the failure, [revert](#reverting-commits) immediately.
+1. Search through the [Flutter issues] for any existing issues with the same error. Some flakes are not specific to a specific test suite, but are nonetheless flakes and should not require a revert.
+1. If the test failure is not a known flake or infrastructure issue, [revert it immediately](#reverting-commits).
+1. When the tree reopens, announce it in the [hackers-ecosystem channel] on [Discord].
+1. If the tree is open, investigate green exclamation point squares, which are tests that have failed, rerun, and then passed. They may be [flaky and warrant an investigation](#handling-a-flaky-test). They also may have hit an intermittent infrastructure issue.
+
+Unmute the [hackers-ecosystem channel] and [hackers-infra channel] on [Discord]. Contributors are encouraged to escalate tree closures to you. Respond there as quickly as possible.
+
+### Rollers
+
+Check that all of the auto-rollers are running:
+* [flutter/flutter → flutter/packages][flutter-to-packages roller]
+* [flutter/packages → flutter/flutter][packages-to-flutter roller]
+    * This roller only rolls once per day, on weekdays, in order to avoid roller loops with the flutter/flutter → flutter/plugin roller.
```

Comment:
```suggestion
    * This roller only rolls once per day, on weekdays, in order to avoid roller loops with the flutter/flutter → flutter/packages roller.
```